installer/roles/local_docker/tasks/standalone.yml

```diff
@@ -104,6 +104,7 @@
       DATABASE_PASSWORD: "{{ pg_password }}"
       DATABASE_PORT: "{{ pg_port }}"
       DATABASE_HOST: "{{ pg_hostname_actual }}"
+      DATABASE_SSLMODE: "{{ pg_sslmode }}"
```

Comment:
Hi! Thanks for this. Just one small thing.. I think it would be best to change these to: 

```
{{ pg_sslmode | default(omit) }}
```

Reply:
Adopted the omit filter in 0729190.